tests/eventgrid_functions/ping/function.json

```diff
@@ -0,0 +1,12 @@
+{
+  "scriptFile": "main.py",
+  "disabled": false,
+  "bindings": [
+    {
+      "authLevel": "anonymous",
```

Comment:
Can you open an issue to track figuring out testing with different autheLevels? anonymous is not the default.  Eventually, we need to remove "authLevel": "anonymous" from all the http tests